include/torch-mlir/Dialect/TorchConversion/Transforms/Passes.h

```diff
@@ -28,7 +28,19 @@ void createTorchBackendToLinalgOnTensorsBackendPipeline(OpPassManager &pm);
 
 /// Creates a pipeline that lowers from the torch backend contract to the
 /// TOSA backend contract.
-void createTorchBackendToTosaBackendPipeline(OpPassManager &pm);
+struct TosaBackendPipelineOptions
+    : public PassPipelineOptions<TosaBackendPipelineOptions> {
+  // If this option is true, custom complex operations.
+  // If this option is false, skip decomposition of complex operations.
+  Option<bool> custom{*this, "custom-ops",
+                      llvm::cl::desc("custom complex operations."),
+                      llvm::cl::init(true)};
+  ListOption<std::string> customOps{
+      *this, "custom-ops",
+      llvm::cl::desc("List of ops to be converted to the backend."),
+      llvm::cl::ZeroOrMore};
```

Comment:
I think the build issue is here. There are two options both with the same name `"custom-ops"`

Reply:
Make sense. Thanks, Ramiro. I have this fixed, could you review and approved this patch if everything else makes sense?